public/video-ui/src/actions/UploadActions/getUploads.js

```diff
@@ -10,8 +10,6 @@ function runningUploads(uploads) {
 }
 
 function errorGettingUploads(error) {
-  Logger.error(error);
```

Comment:
We now print useful info about the error into the big red UI box rather than on the console